apis/compute/v1alpha1/types.go

```diff
@@ -151,3 +151,128 @@ func (kc *KubernetesClusterClassList) GetPortableClassItems() []resource.Portabl
 	}
 	return items
 }
+
+// VirtualMachineSpec specifies the desired state of a VirtualMachine.
+type VirtualMachineSpec struct {
+	runtimev1alpha1.ResourceClaimSpec `json:",inline"`
+}
+
+var _ resource.Claim = &VirtualMachine{}
+
+// +kubebuilder:object:root=true
+
+// A VirtualMachine is a portable resource claim that may be satisfied by
+// binding to a Virtual Machine managed resource such as an AWS EC2 instance
+// or an Azure VM.
+// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.bindingPhase"
+// +kubebuilder:printcolumn:name="CLUSTER-CLASS",type="string",JSONPath=".spec.classRef.name"
```

Comment:
`CLUSTER-CLASS` and `CLUSTER-REF` are forgotten here I guess

Reply:
Good catch! Fixed now.